docs/index.md

```diff
@@ -1,91 +1,121 @@
 # Coyote
 
-Can you find the deadlock bug in this BoundedBuffer class?
-
-```c#
-public class BoundedBuffer
-{
-    private readonly object SyncObject = new object();
-    private readonly object[] Buffer;
-    private int PutAt;
-    private int TakeAt;
-    private int Occupied;
-
-    public BoundedBuffer(int bufferSize)
-    {
-        this.PulseAll = pulseAll;
-        this.Buffer = new object[bufferSize];
+## What is Coyote?
```

Comment:
Should we remove this heading, as there is a `Coyote` heading already directly above this, now that it became the home page? Then users can dive directly into the text!

Reply:
Oh, I think you just changed this to `Welcome to Coyote`.

Reply:
yep